### DISCUSSION TITLE
useFirestore: use auth info?

### DISCUSSION THREAD

**Discussion by auctumnus** [vueuse/vueuse#1498] [Q&A]
Hi! I'm trying to use the `useFirestore` and `useAuth` together, but I can't seem to find a way to use user information from `useAuth` in the document reference for `useFirestore`.

This code, for example, doesn't work:
```js
const { isAuthenticated, user } = useAuth(auth)
const todos = useFirestore(db.collection('todos').where('user', '==', user.uid))
```

because `user` is still `null` by the time that the docref is created. Is there any (idiomatic) way to go about this?

**Comment by dosstx** [comment]
I am also interested in hearing more about this.
